### PR TITLE
Leads support

### PR DESCRIPTION
**Pull request by rohanfyle**
image

image

### CODE REVIEW

File: test/contact.json [hunk 1]
```diff
@@ -15,6 +15,7 @@
       "id": 0,
```
Comment (rohanfyle) [inline test/contact.json]: I have added some missing properties because of which the tests were failing

File: test/account.json [hunk 2]
```diff
@@ -38,6 +38,12 @@
       "cf_no_of_employees_range": "xxx",
```
Comment (rohanfyle) [inline test/account.json]: I have added some missing properties because of which the tests were failing

File: README.md [hunk 3]
```diff
@@ -71,6 +71,18 @@ deals = list(fs.deals.get_all_generator(view_id=view_id))
 # get single deal
 deal_id = 12121
 deal = fs.deals.get(id=deal_id)
+
+# get lead views
+leads = fs.leads.get_views()
+
+# get leads in a view
+view_id = 1231
```
Comment (snarayanank2) [inline README.md]: can you update the example to pick a view id from list of views from previous call?

File: README.md [hunk 4]
```diff
@@ -71,6 +71,18 @@ deals = list(fs.deals.get_all_generator(view_id=view_id))
 # get single deal
 deal_id = 12121
 deal = fs.deals.get(id=deal_id)
+
+# get lead views
+leads = fs.leads.get_views()
```
Comment (snarayanank2) [inline README.md]: should be lead_views = fs.leads.get_views()

**Pull request merged**